content/soarian/dstu2/diagnostic/observation.md

```diff
@@ -41,28 +41,28 @@ _Implementation Notes_
 
 * The API returns the most current charted Observation list. 
 
-* The Narrative includes additional information pertaining to the observation, and should be shown to the user to ensure completeness of clinical content and context
+* The API returns values in reverse chronological order by the most currently charted date.
 
-### Parameters 
-
-|Name |Required? | Type | Description
-| --- | --- | --- | --- |
-| patientId | Y | [reference](http://hl7.org/fhir/DSTU2/search.html#reference) | The patient identifier provided in a pre-requisite authorization step. |
-| date | N | [date](http://hl7.org/fhir/DSTU2/search.html#date) as adjusted per implementation notes | Null or specific date or a date range. |
-
-Notes:
+* The API returns charted content that is not marked as erroneous.
```

Comment:
Yes. you are correct. we have verified the API and we are returning the status as 'final'. I have updated the document as 'The API returns charted content in final status.'